src/components/UpcomingShiftCard/UpcomingShiftCard.jsx

```diff
@@ -23,38 +41,33 @@ const styleDate = (data) => {
       'November',
       'December',
    ];
-
-   month = months[parseInt(month, 10)];
-
-   return `${month} ${day}, ${year}`;
-};
-
-const styleTime = (data) => {
-   const time = data.split(' ')[1];
-   const hours = time.split(':')[0];
-   const min = time.split(':')[1];
-   return `${hours}:${min}`;
+   return (
+      <Card>
+         <Card.Body>
+            <div>
+               <Card.Title>
+                  <Header1>
+                     {DOW[df.getDay()]}, {months[df.getMonth()]} {df.getDate()}
+                  </Header1>
+               </Card.Title>
+               <Card.Text>
+                  {df.toLocaleString('en-US', {
+                     hour: 'numeric',
+                     minute: 'numeric',
+                     hour12: true,
+                  })}
+                  -&nbsp;
+                  {dt.toLocaleString('en-US', {
+                     hour: 'numeric',
+                     minute: 'numeric',
+                     hour12: true,
+                  })}
+               </Card.Text>
+            </div>
+            <Card.Link href="#">
+               <StyledText>Details &gt;</StyledText>
```

Comment:
Is there a reason you removed the chevron?

Reply:
Some files used the chevron others used the > symbol, just wanted to maintain consistency. I can replace all with the chevron if you prefer that.

Reply:
Yea I would just replace with the chevron, if you could let me know what other files use the > symbol that would be great.